src/oatpp/core/async/Executor.hpp

```diff
@@ -63,9 +64,9 @@ class Executor {
 
     oatpp::async::Processor& getProcessor();
 
-    void pushTasks(utils::FastQueue<CoroutineHandle>& tasks) override;
+    void pushTasks(utils::FastQueue<CoroutineHandle>& tasks) override GPP_ATTRIBUTE(noreturn);
```

Comment:
Doesn't look like a `noreturn` function to me.
Please give more details.

Reply:
The implementation throws a std::runtime_error